modules/router/manifests/nginx.pp

```diff
@@ -76,9 +58,6 @@
   $check_requests_warning = '@25',
   $check_requests_critical = '@10',
   $robotstxt = '',
-  $proxy_busy_buffers_size = '8k',
-  $proxy_buffers = '8 4k',
-  $proxy_buffer_size = '4k',
```

Comment:
I'm not sure why these are being removed? I assume they should be increased in size.

These don't use the proxy-vhost.conf file and instead are put into: https://github.com/alphagov/govuk-puppet/blob/ab4c01f94cdac00c0c5e286a30143748de099266/modules/router/templates/router_include.conf.erb#L94-L96

Reply:
have updated the router side of things and added these back in